css/main.scss

```diff
@@ -628,7 +595,7 @@ header .watch > a:hover {
 .timeline > h3.working-on::after,
 .timeline > ul.working-on > li::before,
 .timeline > ul.working-on > li::after {
-   border-color: $color-RO_donkerblauw;
+    border-color: $color-RO_donkerblauw;
```

Comment:
Nitpicky, maar misschien kunnen we deze variabele-namen qua kleur ook vertalen naar Engels, het is nu een beetje een gekke mix :)

Reply:
Dit had ik ook meegepakt, zie commits